crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -1,12 +1,16 @@
 use cairo_felt::Felt252;
+use conversions::byte_array::ByteArray;
 use regex::Regex;
 
 #[must_use]
 pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt252>> {
-    let re = Regex::new(r"Got an exception while executing a hint: Hint Error: Execution failed. Failure reason:\s\w*\s\(\'(.*)\'\)\.")
+    let re_felt_array = Regex::new(r"Got an exception while executing a hint: Hint Error: Execution failed\. Failure reason:\s\w*\s\('(.*)'\)\.")
         .expect("Could not create panic_data matching regex");
 
-    if let Some(captures) = re.captures(err) {
+    let re_string = Regex::new(r#"(?s)Got an exception while executing a hint: Hint Error: Execution failed\. Failure reason: "(.*)"\."#)
```

Comment:
```suggestion
    let re_string = Regex::new(r#"Got an exception while executing a hint: Hint Error: Execution failed\. Failure reason: "(.*)"\."#)
```
is this needed?

Reply:
Let's add tests for this later

Reply:
@drknzz it enables us to parse multiline stuff which is pretty important

Reply:
I added tests and revamped previous regex